workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Readers/BlittableReaderBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Unity.Entities;
+
+namespace Improbable.Gdk.Core
```

Comment:
Why isn't this in a sub-namespace, or better yet, not in Core at all?

Reply:
Yeah please put this in `Improbable.Gdk.Core.Monobehaviours` or similar